static/app/components/draggableTabs/draggableTabList.tsx

```diff
@@ -268,7 +269,6 @@ const TabListWrap = styled('ul', {
   margin: 0;
   list-style-type: none;
   flex-shrink: 0;
-  padding-left: 15px;
```

Comment:
This padding was here to make the stories component match the figma designs, but this padding is already accounted for in the issue stream. 

<img width="287" alt="image" src="https://github.com/user-attachments/assets/cc7db8ae-4701-4650-9432-86fb836adbf6">